server/src/test/java/org/opensearch/action/admin/cluster/reroute/ClusterRerouteRequestTests.java

```diff
@@ -161,7 +161,7 @@ public void testEqualsAndHashCode() {
             assertEquals(request, copy);
             assertEquals(request.hashCode(), copy.hashCode());
 
-            // Changing masterNodeTime makes requests not equal
+            // Changing clusterManagerNodeTime makes requests not equal
```

Comment:
`masterNodeTimeout` -> `clusterManagerNodeTimeout`

Reply:
[masterNodeTimeout](https://github.com/opensearch-project/OpenSearch/blob/2.0.0-rc1/server/src/main/java/org/opensearch/action/support/master/MasterNodeRequest.java#L68) is a public method in `sever/src/main` directory. I will address them in a following PR and target for version 3.0
But the `masterNodeTime` in the comment seems lack a word `out`, I will add it. 😁

Reply:
Updated in the commit https://github.com/opensearch-project/OpenSearch/pull/2520/commits/a87cede9b88976e0a043e7ea8670d3de2f400c1e